client/material/portal.css

```diff
@@ -5,6 +5,7 @@
 .portal {
   contain: layout style;
   position: relative;
+  z-index: 0;
```

Comment:
probably wanna comment about stacking here as well (like you did in the global CSS)